jerry-core/debugger/debugger.c

```diff
@@ -815,6 +815,22 @@ jerry_debugger_send_parse_function (uint32_t line, /**< line */
 } /* jerry_debugger_send_parse_function */
 
 /**
+ * Send the output of the program to the debugger client.
+ * Currently only sends print output.
+ */
+void
+jerry_debugger_send_output (jerry_char_t buffer[], /**< buffer */
+                            jerry_size_t str_size, /**< string size */
+                            uint8_t type) /**< type of output */
+{
+  if (JERRY_CONTEXT (debugger_flags) & JERRY_DEBUGGER_CONNECTED)
+  {
+    const uint8_t *d_char = (uint8_t *) buffer;
+    jerry_debugger_send_string (JERRY_DEBUGGER_OUTPUT_RESULT, type, d_char, sizeof (char) * str_size);
```

Comment:
This type cast could be the part of `jerry_debugger_send_string`. Separate each argument to one line.